packages/h/highlight.js.json

```
@@ -1,22 +1,23 @@
 {
   "name": "highlight.js",
   "filename": "highlight.min.js",
-  "description": "Syntax highlighting with language autodetection.",
+  "description": "Syntax highlighting with language auto-detection.",
   "homepage": "http://highlightjs.org",
   "keywords": [
     "highlight",
     "syntax highlighter"
   ],
   "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/highlightjs/cdn-release.git",
+    "source": "npm",
+    "target": "@highlightjs/cdn-assets",
     "fileMap": [
       {
-        "basePath": "build",
+        "basePath": "",
         "files": [
           "highlight.min.js",
           "languages/*.min.js",
-          "styles/*"
+          "styles/*",
+          "styles/**/*"
```

```suggestion
          "styles/**/*.css"
```
Unrestricted globstars are dangerous and we try to avoid them by ensuring we have extensions etc. to limit them.